### PR TITLE
Terminology page update to include where used in information

### PR DESCRIPTION
**Pull request by dbojicic**
Updated terminology page to include where used in information:
- content added manually (got a scrip to generate the list but it generates output for all terminology resources, not just those associated with MS)

### CODE REVIEW

File: input/pagecontent/terminology.md [hunk 1]
```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
...
-- [NameUse](https://hl7.org/fhir/R4/valueset-name-use.html)
-- [Observation Category Codes](https://hl7.org/fhir/R4/valueset-observation-category.html)
-- [Observation Interpretation Codes](https://hl7.org/fhir/R4/valueset-observation-interpretation.html)
-- [Observation Reference Range Meaning Codes](https://hl7.org/fhir/R4/valueset-referencerange-meaning.html)
-- [ObservationStatus](http://hl7.org/fhir/R4/valueset-observation-status.html)
-- [Participant type](https://hl7.org/fhir/R4/valueset-encounter-participant-type.html)
-- [Provenance activity type](https://hl7.org/fhir/R4/valueset-provenance-activity-type.html)
-- [Vital Signs Units](https://hl7.org/fhir/R4/valueset-ucum-vitals-common.html)
-- [VitalSigns](https://hl7.org/fhir/R4/valueset-observation-vitalsignresult.html)
+|ValueSet|Profile where used in|Published|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:11]: For 'Profile where used in', suggest 'Profile where ValueSet used'.

File: input/pagecontent/terminology.md [hunk 2]
```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
...
+|---| ---| ---|
+|[ActEncounterCode - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-au-v3-ActEncounterCode-extended.html)|[AU Core Encounter](StructureDefinition-au-core-encounter.html)|AU Base|
+|[AdministrativeGender](https://hl7.org/fhir/R4/valueset-administrative-gender.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|FHIR|
+|[Adverse Reaction Agent](https://healthterminologies.gov.au/fhir/ValueSet/adverse-reaction-agent-1)|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|NCTS|
+|[AllergyIntolerance Clinical Status Codes](https://hl7.org/fhir/R4/valueset-allergyintolerance-clinical.html)|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|FHIR|
+|[AllergyIntolerance Verification Status Codes](http://hl7.org/fhir/R4/valueset-allergyintolerance-verification.html)|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|NCTS|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:17]: Change NCTS to AU Base

File: input/pagecontent/terminology.md [hunk 3]
```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
...
+|[AllergyIntoleranceSeverity](http://hl7.org/fhir/R4/valueset-reaction-event-severity.html)|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|NCTS|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:18]: Change NCTS to AU Base

File: input/pagecontent/terminology.md [hunk 4]
```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
...
+|[Australian Pronouns](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/australian-pronouns-1)|[AU Core Patient](StructureDefinition-au-core-patient.html)|NCTS|
+|[Australian States and Territories](https://healthterminologies.gov.au/fhir/ValueSet/australian-states-territories-2)|[AU Core Patient](StructureDefinition-au-core-patient.html), [AU Core Organization](StructureDefinition-au-core-organization.html), [AU Core Location](StructureDefinition-au-core-location.html)|NCTS|
+|[Body Length Units](http://hl7.org/fhir/ValueSet/ucum-bodylength)|[AU Core BodyHeight](StructureDefinition-au-core-bodyheight.html), [AU Core Waist Circumference](StructureDefinition-au-core-waistcircum.html)|FHIR|
+|[Body Temperature Units](http://hl7.org/fhir/ValueSet/ucum-bodytemp)|[AU Core BodyTemperature](StructureDefinition-au-core-bodytemp.html)|FHIR|
+|[Body Weight Units](http://hl7.org/fhir/ValueSet/ucum-bodyweight)|[AU Core Body Weight](StructureDefinition-au-core-bodyweight.html)|FHIR|
+|[Clinical Condition](https://healthterminologies.gov.au/fhir/ValueSet/clinical-condition-1)|[AU Core Condition](StructureDefinition-au-core-condition.html),|NCTS|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:28]: Remove comma after AU Core Condition (AU Core Condition,) - repeated in line for Condition/Diagnosis Severity; ConditionCategoryCodes; ConditionClinicalStatusCodes; ConditionVerificationStatus.

File: input/pagecontent/terminology.md [hunk 5]
```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
...
+|[Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1)|[AU Core Patient](StructureDefinition-au-core-patient.html)|NCTS|
+|[Healthcare Organisation Role Type](https://healthterminologies.gov.au/fhir/ValueSet/healthcare-organisation-role-type-1)|[AU Core Organization](StructureDefinition-au-core-organization.html)|NCTS|
+|[hl7VS-degreeLicenseCertificate - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-v2-0360-extended.html)|[AU Core Practitioner](StructureDefinition-au-core-practitioner.html)|AU Base|
+|[hl7VS-identifierType - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-au-v2-0203-extended.html)|[AU Core Organization](StructureDefinition-au-core-organization.html)|AU Base|
+|[hl7VS-providerRole - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-au-v2-0443-extended.html)|[AU Core Immunization](StructureDefinition-au-core-immunization.html)|AU Base|
+|[https://hl7.org/fhir/R4/valueset-identifier-use.html](https://hl7.org/fhir/R4/valueset-identifier-use.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|FHIR|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:46]: URL (https://hl7.org/fhir/R4/valueset-identifier-use.html) shown as ValueSet name - change to 'IdentifierUse'

File: input/pagecontent/terminology.md [hunk 6]
```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
...
+|[Participant type](https://hl7.org/fhir/R4/valueset-encounter-participant-type.html)|[AU Core Encounter](StructureDefinition-au-core-encounter.html)|FHIR|
+|[PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-pbs-item.html)|[AU Core MedicationRequest](StructureDefinition-au-core-medicationrequest.html), [AU Core Medication](StructureDefinition-au-core-medication.html)|AU Base|
+|[Practitioner Role](https://healthterminologies.gov.au/fhir/ValueSet/practitioner-role-1)|[AU Core PractitionerRole](StructureDefinition-au-core-practitionerrole.html)|NCTS|
+|[Procedure](https://tx.ontoserver.csiro.au/fhir/ValueSet/procedure-1)|[AU Core Procedure](StructureDefinition-au-core-procedure.html)|NCTS|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:64]: Should the url for Procedure VS be https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/procedure-1?

File: input/pagecontent/terminology.md [hunk 7]
```diff
@@ -83,49 +79,49 @@ This implementation guide does not define any unique code systems. The code syst
...
+|[AllergyIntolerance Clinical Status Codes](https://hl7.org/fhir/R4/valueset-allergyintolerance-clinical.html)|[AllergyIntolerance Clinical Status Codes](https://hl7.org/fhir/R4/valueset-allergyintolerance-clinical.html)|FHIR|
+|[AllergyIntolerance Verification Status](https://hl7.org/fhir/R4/codesystem-allergyintolerance-verification.html)|[AllergyIntolerance Verification Status Codes](http://hl7.org/fhir/R4/valueset-allergyintolerance-verification.html)|FHIR|
+|[AllergyIntoleranceSeverity](https://hl7.org/fhir/R4/codesystem-reaction-event-severity.html)|[AllergyIntoleranceSeverity](https://hl7.org/fhir/R4/valueset-reaction-event-severity.html)|FHIR|
+|[Australian Immunisation Register Vaccine](https://www.servicesaustralia.gov.au/air-vaccine-code-formats?context=20)|[Australian Immunisation Register Vaccine](https://healthterminologies.gov.au/fhir/ValueSet/australian-immunisation-register-vaccine-1)|NCTS|
+|[Australian Indigenous Status](https://healthterminologies.gov.au/fhir/CodeSystem/australian-indigenous-status-1)|[Australian Indigenous Status](https://healthterminologies.gov.au/fhir/ValueSet/australian-indigenous-status-1)|AU Base|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:91]: Change from AU Base to NCTS

File: input/pagecontent/terminology.md [hunk 8]
```diff
@@ -83,49 +79,49 @@ This implementation guide does not define any unique code systems. The code syst
...
+|[DegreeLicenseCertificate AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-v2-0360.html)|[hl7VS-degreeLicenseCertificate - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-v2-0360-extended.html)|AU Base|
+|[EncounterStatus](https://hl7.org/fhir/R4/codesystem-encounter-status.html)|[EncounterStatus](https://hl7.org/fhir/R4/valueset-encounter-status.html)|FHIR|
+|[EventStatus](https://hl7.org/fhir/R4/codesystem-event-status.html)|[EventStatus](https://hl7.org/fhir/R4/valueset-event-status.html)|FHIR|
+|[IdentifierType AU](https://build.fhir.org/ig/hl7au/au-fhir-base//CodeSystem-au-v2-0203.html)|[DVA Entitlement](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-dva-entitlement.html), [hl7VS-identifierType - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-v2-0203-extended.html)|FHIR|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:100]: Change from FHIR to AU Base

File: input/pagecontent/terminology.md [hunk 9]
```diff
@@ -83,49 +79,49 @@ This implementation guide does not define any unique code systems. The code syst
...
+|[identifierType](htts://terminology.hl7.org/5.3.0/CodeSystem-v2-0203.html)|[hl7VS-identifierType - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-v2-0203-extended.html)|FHIR|
```
Comment (JayMurdoch) [inline input/pagecontent/terminology.md:101]: Duplicate of IdentifierType AU above.

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
Hi Danielle,
See changes in comments.
Thanks.

File: input/pagecontent/terminology.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:17]: fixed

File: input/pagecontent/terminology.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:18]: fixed

File: input/pagecontent/terminology.md (hunk 8 above)
Comment (dt-r) [reply input/pagecontent/terminology.md]: fixed

File: input/pagecontent/terminology.md (hunk 9 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:101]: can't spot the duplicate. I can only see the two Identifier type code systems, one is AU and the other if FHIR. You might have to copy and paste the fragement.

File: input/pagecontent/terminology.md (hunk 4 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:28]: fixed

File: input/pagecontent/terminology.md (hunk 5 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:46]: Not sure what you meant by duplicate. The code system was incorrectly hyperlinked to the value set so I've swapped that out.

File: input/pagecontent/terminology.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/terminology.md]: Changed to "Profile where used" to be consistent with wording choices in CodeSystem table in AU Core.

File: input/pagecontent/terminology.md (hunk 6 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:64]: the url for healthterminologies artefacts should be the canonical, so it should be https://healthterminologies.gov.au/fhir/ValueSet/procedure-1

I've made that change in two places.

File: input/pagecontent/terminology.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:17]: Can't see the change - however I made an error in my review comment. Change from NCTS to FHIR, sorry.

File: input/pagecontent/terminology.md (hunk 3 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:18]: Can't see the change - however I made an error in my review comment. Change from NCTS to FHIR, sorry.

File: input/pagecontent/terminology.md (hunk 5 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:46]: I see two entries for IdentifierUse - the first has the ValueSet name as the URL (line 46), and the second entry has the name 'IdentifierUse' (line 47).

File: input/pagecontent/terminology.md (hunk 9 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:101]: Apologies - I see the difference. Thanks.

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
Hi Danielle, 3 comments to cover off. Thanks.

File: input/pagecontent/terminology.md (hunk 7 above)
Comment (dt-r) [reply input/pagecontent/terminology.md]: fixed

File: input/pagecontent/terminology.md (hunk 9 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:101]: Don't forget to resolve a convo when you are satisfied with the change :)

File: input/pagecontent/terminology.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:18]: fixed. resolve when satisfied with change.

File: input/pagecontent/terminology.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:17]: fixed. resolve when satisfied with change.

File: input/pagecontent/terminology.md (hunk 5 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:46]: removed he entry that had the value set url as the text to be displayed.

**Review by JayMurdoch — APPROVED** [review]
Review complete, no further issues.

**Pull request merged**